src/tools/pub/pubspec.md

```diff
@@ -410,6 +414,37 @@ Each download of the package includes all screenshot files.
 Pub.dev generates the package's thumbnail image from the first screenshot. If 
 this screenshot uses animation, pub.dev uses its first frame.
  
+### Topics
+
+Package authors can use the `topics` field to categorize their package. Topics
+can be used to assist discoverability during search with filters on pub.dev.
+Pub.dev displays the topics on the package page as well as in the search
+results.
+
+The field consists of a list of names. For example:
+
+```yaml
+topics:
+  - network
+  - http
+```
+
+Pub.dev limits topics to the following specifications:
+
+- At most 5 topics per package.
+- The length of the topic name should be 2-32 characters.
+- The topic name should consist of lowercase alphanumeric characters or dash
+  (but no double dash), starting with a-z and ending with a-z or 0-9.
+
+
```

Comment:
**suggestion:** Add a heading for "Best Practices".
```markdown
#### Best Practices
```

Reply:
done

Reply:
We don't do this for other subjects on the page.

We could put it in an "Note" block, but I think that's overkill.